frontend/src/components/worksheets/Worksheet.js

```diff
@@ -318,6 +336,22 @@ class Worksheet extends React.Component {
             }.bind(this),
         );
 
+        // Show/hide web terminal (action bar)	
+        Mousetrap.bind(	
```

Comment:
I just added in the code that was there initially ¯\_(ツ)_/¯ -- will take a look though

Reply:
You were right -- forgot to add in the functionality for showing/hiding the action bar. I've added it now -- it turned out to be a bit more complicated than originally implemented because of the <Sticky> component on the header. Please take a look!